lib/ecto_adapters_dynamodb.ex

```diff
@@ -14,30 +14,19 @@ defmodule Ecto.Adapters.DynamoDB do
 
   use Bitwise, only_operators: true
 
-  alias ExAws.Dynamo
+  alias Ecto.Adapters.DynamoDB.Cache
+  alias Ecto.Adapters.DynamoDB.RepoConfig
   alias Ecto.Query.BooleanExpr
+  alias ExAws.Dynamo
 
   @pool_opts [:timeout, :pool_size, :migration_lock]
 
   # DynamoDB will reject attempts to batch write more than 25 records at once
   # https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html
   @batch_write_item_limit 25
 
-  def start_link({_module, config}) do
```

Comment:
Why was this no longer necessary (or not necessary in the first place)?

Reply:
It never was. I'm not sure what it was doing - the Agent process it launched was never referenced again.